ktext.md

```diff
@@ -402,6 +443,7 @@ Most laptop keyboards are PS2! You will want to grab VoodooPS2 even if you have
 
 * [ECEnabler](https://github.com/1Revenger1/ECEnabler/releases)
   * Fixes reading battery status on many devices (Allows reading EC fields over 8 bits long)
+  * Supports OS X 10.7 and above
```

Comment:
Afaik, it's not actually needed in 10.4-10.6. May be worth adding a note about this.